go.mod

```diff
@@ -412,14 +420,12 @@ require (
 
 require (
 	github.com/containerd/containerd v1.6.1 // indirect
-	github.com/klauspost/compress v1.15.1 // indirect
+	github.com/klauspost/compress v1.15.4 // indirect
 	github.com/shirou/gopsutil/v3 v3.22.4
 	github.com/spf13/cobra v1.4.0 // indirect
 	golang.org/x/sys v0.0.0-20220503163025-988cb79eb6c6
 )
 
-replace github.com/open-telemetry/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.50.0 => github.com/observiq/opentelemetry-collector-contrib/exporter/googlecloudexporter v0.0.0-20220504030020-bb60df1ad6e0
-
```

Comment:
Same here; Does logs support still work for the googlecloudexporter?

Reply:
fixed the dependency on the newest commits of our fork of the exporter